bogota-july-2018/javascript-calculator/Bryan Murillo/calculadora Day5.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">    
+    <meta name="author" content="Bryan Murillo">
+    
+    <title>Calculadora</title>
+    <script type="text/javascript">
+        var op = {
+            '+': function(a,b) {return a+b},
+            '-': function(a,b) {return a-b},
+            '*': function(a,b) {return a*b},
+            '/': function(a,b) {return b !== 0 ? a/b : "valor 0"}
+        }
+        
+        function calcular() {
+            let a = Number(document.getElementById("a").value);
+            let b = Number(document.getElementById("b").value);
+            
+            let opera = document.getElementById("operation").value
+            console.log(op[opera](a,b));
```

Comment:
You are not even displaying in the UI the result

Reply:
I'm sorry, this is not the UI calculator yet, this is the homework for Day 5, the UI calculator is the homework of Day 6.

Reply:
Oh, i've assumed that with an html file with some UI